### DISCUSSION TITLE
The dynamic import usage creates  "maximum update depth exceeded" if rendered as list.

### DISCUSSION THREAD

**Discussion by Refrezsh** [vercel/next.js#50108] [App Router]
I have already migrated the app to App router, and ran into problems when using "dynamic".

1. If I use "dynamic" in a component which is rendered in the list, I get "maximum update depth exceeded". I have a filter component that, depending on the cms configuration, can dynamically import either a checkbox, or a tile, or a tag component. If I render such component more than 40-50 times, there will be an error. When disabling dynamic the error disappears. This problem did not occur in the pages.

2. Dynamic component import is noticeably slower than in Pages router. On average, 400 ms, which can be felt. I use this import on some components and icons that I don't want to embed in html.

I solved the first problem by disabling dynamic in those components that can render a list. Would like to know why the dynamic import has become slower ?

**Comment by rc-markzhou** [comment]
had the same issue
